CacheMultimap.adoc

```diff
@@ -0,0 +1,79 @@
+= Native cache Mutimap
+
+The goal is to provide Infinispan Native CacheMultimap support. The proposal here is to start with the smallest interface implementation, in order to provide a proper API to the first multimap client, vert-x project.
+However, this API will evolve, and this first implementation must work on Embedded and Server mode.
+This first Multimap won't support duplicate values on the same key. 
+
+[source, java]
+.CacheMultimap.java
+----
+public interface CacheMultimap<K, V> {
+   boolean put(K key, V value); // <1> <2>
+
+   Collection<V> get(K key); // <3>
+
+   boolean remove(K key); //<4>
```

Comment:
@karesti looking again on the return value of `Map.compute()`, if you want to utilize this for the multimap, you need to use `void put()` and `void remove()`, since the `remappingFunction` can't return one value and store another - `compute()` would return the new value and seemingly you need to use `cache.withFlags(Flag.IGNORE_RETURN_VALUE).compute(key, addEntryToCollectionFunction)`.
Can you see something else that could make 2. or 3. work?